README.md

```diff
@@ -20,7 +21,7 @@ Tests are done by testing the sample apps that use this.
 
 ---
 
-Developed using Python 3.9.1.
+Developed using Python 3.7.9.
```

Comment:
i think you can keep this as it is, since you probably were using 3.9.1 and just have this back testing manually?